numpy/lib/nanfunctions.py

```diff
@@ -755,35 +756,34 @@ def nanmedian(a, axis=None, out=None, overwrite_input=False, keepdims=False):
     a : array_like
         Input array or object that can be converted to an array.
     axis : int, optional
-        Axis along which the medians are computed. The default (axis=None)
-        is to compute the median along a flattened version of the array.
-        A sequence of axes is supported since version 1.9.0.
+        Axis along which the medians are computed. The default
+        (axis=None) is to compute the median along a flattened version
```

Comment:
Hmm. if `None` is the default should have `axis : {int, None}, optional`.  Could then omit `(axis=None)`.